src/cml.js

```diff
@@ -473,6 +473,9 @@ class CML {
     }
 
     const title = `CML PR for ${target} ${shaShort}`;
+    if (skipCI) {
```

Comment:
```suggestion
    if (this.driver === GITLAB && skipCI) {
```

Reply:
A counter suggestion: rather than having this if statement in the "main", what about passing the skipCI variable to the drivers? Then each driver can choose what to do. E.g. GitLab can append `[skip ci]` to the title.

Reply:
yes, definitely sounds better...